packages/expo/src/DangerZone.ts

```diff
@@ -51,4 +51,7 @@ module.exports = {
   get ScreenStack() {
     return require('react-native-screens').ScreenStack;
   },
+  get firebase() {
```

Comment:
Is this intentionally uncapitalized?

Reply:
Yes, it's to match the `firebase` API. Unfortunately this makes it different to the other modules.
```js
import firebase from 'firebase';

import firebase from 'react-native-firebase';

import firebase from 'expo-firebase-app';

import { DangerZone } from 'expo'
const { firebase } = DangerZone;
```